lib/NUPACK/emscripten/FullEnsemble.cpp

```diff
@@ -180,20 +178,23 @@ FullAdvancedResult* FullEnsembleNoBindingSite(const std::string& seqString, int
     }
 
     //initialize the result to return
+    
     FullAdvancedResult* result = new FullAdvancedResult();
     
     std::string mfeStructure;
     //get dot bracket notation from data 
     for ( int i = 0; i < suboptStructs.nStructs; i++ ) {
         oneDnaStruct currentStruct = suboptStructs.validStructs[i];
-
+        
         //get the secondary strucutre in dot paren notation 
-        std::string singlestructure = getDotParens(pseudoknotted, suboptStructs.seqlength, &currentStruct);
+        //old method
+        //std::string singlestructure = getDotParens(pseudoknotted, suboptStructs.seqlength, &currentStruct);
         
+        std::string singlestructure = getDotParens(pseudoknotted, suboptStructs.seqlength, &currentStruct);
         if (i == 0)
         {
           //this is the first one so it is the mfe          
-          mfeStructure =  singlestructure;
+          mfeStructure =singlestructure;
```

Comment:
Nit: unnecessary whitespace change

Reply:
changed back as it is better with the space anyways